src/components/Progress/Progress.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Progress } from 'reactstrap';
+
+import './Progress.scss';
+
+export function ASABProgress({
+	value = 0,
+	color = 'primary',
+	animated = true,
+	showPercentage = true,
+	className = '',
+	children,
+	...props
+}) {
+	// Ensure value is a number
+	let progressValue = Number(value);
```

Comment:
You need to ensure that `props.value` exist and that its a valid number, otherwise the app can crash because of it.